extensions/2.0/Vendor/DRACO_animation_compression/README.md

```diff
@@ -0,0 +1,236 @@
+# Draco_animation_compression
+
+## Contributors
+
+* Fan Zhang, Google, <mailto:zhafang@google.com>
+* Michael Hemmer, <mailto:hemmer@google.com>
+* Ondrej Stava, Google, <mailto:ostava@google.com>
+* Frank Galligan, Google, <mailto:fgalligan@google.com>
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines a schema to use [Draco compression library (non-normative)](https://github.com/google/draco) to compress `animations` in glTF format. This allows glTF to support streaming compressed animation data instead of the raw data.
+
+The [conformance](#conformance) section specifies what an implementation must do when encountering this extension, and how the extension interacts with the attributes defined in the base specification.
+
+## glTF Schema Updates
+
+If a Draco compressed version of one or more `animations` is provided then `Draco_animation_compression` must be added to `extensionsUsed`.
+
+```javascript
+"extensionsUsed" : [
+    "Draco_animation_compression"
+]
+```
+
+If the uncompressed version of one or more `animations` are not provided, then `Draco_animation_compression` must be added to `extensionsRequired`.
+```javascript
+"extensionsRequired" : [
+    "Draco_animation_compression"
+]
+```
+
+If the root object of a glTF asset contains `extensions` property and the `extensions` property defines its `Draco_animation_compression` property, then the Draco animation compression must be used. If there are any compressed animations then the `Draco_animation_compression` must define the `compressedAnimations` property. The `compressedAnimations` property consists of an array of compressed animation objects. Each element in `compressedAnimations` must have the following properties:
+
+#### input
+The `input` property defines a single `input` of a compressed animation. The
+`input` corresponds to a `samplers` element's `input`.
+
+### outputs
+The `outputs` property defines an array of `output` of sharing the same `input`.
+Each element of `outputs` corresponds to a `samplers` element's `output`.
```

Comment:
Are there any constraints on the types of the input/output accessors? Example code shows only Float32, but I assume the type would actually be read from the accessor or the bitstream?